### PR TITLE
Add SUMA 5.0 Extensions and fix the activation keys

### PR DESCRIPTION
**Pull request by mcalmer**
## What does this PR change?

For SUMA + Proxy 5.0, mirror SLE-Micro and the extensions
Also change the activation key product selection 

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- No tests: already covered

- [x] **DONE**

## Links

Issue(s): #
Port(s): # **add downstream PR(s), if any**

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Comment by mcalmer** [comment]
Testrun: https://ci.suse.de/view/Manager/view/Manager-Head/job/manager-Head-dev-acceptance-tests-NUE/5992/

**Review by srbarrios — APPROVED** [review]
LGTM. Thank you Michael.

File: testsuite/features/reposync/srv_sync_products.feature [hunk 1]
```diff
@@ -110,41 +110,49 @@ Feature: Synchronize products in the products page of the Setup Wizard
...
-    And I open the sub-list of the product "Basesystem Module 15 SP4 x86_64"
-    And I select "Containers Module 15 SP4 x86_64" as a product
-    Then I should see the "Containers Module 15 SP4 x86_64" selected
+    And I enter "SUSE Linux Enterprise Micro 5.5" as the filtered product description
+    When I open the sub-list of the product "SUSE Linux Enterprise Micro 5.5 x86_64"
+    And I select "SUSE Manager Proxy Extension 5.0 x86_64" as a product
+    Then I should see the "SUSE Manager Proxy Extension 5.0 x86_64" selected
+    When I click the Add Product button
+    And I wait until I see "Selected channels/products were scheduled successfully for syncing." text
+    And I wait until I see "SUSE Manager Proxy Extension 5.0 x86_64" product has been added
```
Comment (srbarrios) [inline testsuite/features/reposync/srv_sync_products.feature:139]: Indeed, we missed that part now, my bad, when I started all this implementation I wasn't aware of this extension (or maybe it did not exist yet).

**Review by srbarrios — APPROVED** [review]

File: testsuite/features/reposync/srv_sync_products.feature (hunk 1 above)
Comment (mcalmer) [reply testsuite/features/reposync/srv_sync_products.feature:139]: It was missing on our mirrors. Now they are mirrored and visible

**Comment by srbarrios** [comment]
Feel free to merge it, so we have results tomorrow, if by chance you pass through here today

**Comment by mcalmer** [comment]
We pass Core now, but in init-clients the host of the proxy seems to not configured correctly and is not showing up.
There is nothing configured in venv-salt-minion configuration.
That's for another PR

**Pull request merged**